tutorial/explicit/Solver_I.m

```diff
@@ -32,20 +32,24 @@
 if interface.requiresInitialData()
     interface.writeScalarData(meshName, DataNameI, vertex_ID, I0);
 end
-dt = interface.initialize();
+interface.initialize();
+dt = interface.getMaxTimeStepSize();
 
 % Start simulation
 t = t0 + dt;
-while t < t_max;
+while t < t_max
```

Comment:
Why is this change necessary? Something related to MATLAB versions?

Reply:
My linter was saying this was unnecessary. But it does not change anything here, should I remove it or make it consistent across all solvers?

Reply:
Making it consistent would be the way to go :+1:

Reply:
Nevermind, I just checked and I already removed it everywhere